test/views/PensionSchemeViewSpec.scala

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import base.SpecBase
+import forms.PensionSchemeForm
+import org.jsoup.Jsoup
+import play.api.mvc.Call
+import play.api.test.FakeRequest
+import play.api.test.Helpers.running
+import viewmodels.models.PensionSchemeViewModel
+import viewmodels.DisplayMessage
+import views.html.PensionSchemeView
+import org.jsoup.Jsoup
+import play.api.mvc.Call
+
+
+class PensionSchemeViewSpec extends SpecBase {
```

Comment:
More tests needed in this file to cover success and failure cases

Reply:
Added a success and failure case 👍